backend/apps/billing/views.py

```
@@ -1,89 +1,277 @@
+                    {
+                        "price": plan.stripe_price_id,
+                        "quantity": 1,
+                    }
+                ],
+                mode="subscription",
+                success_url=settings.FRONTEND_URL + "/settings/billing?success=true",
+                cancel_url=settings.FRONTEND_URL + "/settings/billing?canceled=true",
+                metadata={"user_id": request.user.id, "plan_id": plan.id},
+            )
+            return Response({"checkout_url": session.url})
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)